### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by santosh1994**
Include support for Navigation.

### CODE REVIEW

**Comment by santosh1994** [comment]
@rahulcs @floydpraveen Please review

File: unbxdSearch.js [hunk 1]
```diff
@@ -580,9 +583,15 @@ var unbxdSearchInit = function(jQuery, Handlebars){
...
-      if((this.options.type == "search" && "query" in this.params && this.params["query"].trim().length > 0) ||
-	 (this.options.type == "browse" && "categoryId" in this.params && this.params["categoryId"].trim().length > 0))
-	this.callResults(this.paintResultSet);
+      if (
+        (this.options.type == 'search' &&
+          'query' in this.params &&
+          this.params['query'].trim().length > 0) ||
+        ((this.options.type == 'browse' || this.options.type === 'category') &&
+          'categoryId' in this.params &&
+          this.params['categoryId'].trim().length > 0)
```
Comment (rahulcs) [inline unbxdSearch.js]: Create smaller functions for these checks?

Comment (santosh1994) [inline unbxdSearch.js]: done

File: unbxdSearch.js [hunk 2]
```diff
@@ -1031,6 +1055,9 @@ var unbxdSearchInit = function(jQuery, Handlebars){
 	url += '&'+ this.options.searchQueryParam +'='+ encodeURIComponent(this.params.query);
       }else if(this.options.type == "browse" && this.params['categoryId'] != undefined){
 	url += '&category-id=' + encodeURIComponent(this.params.categoryId);
+      } else if(this.options.type === "category"
```
Comment (rahulcs) [inline unbxdSearch.js]: Create `isTypeCategory` and use it in `isTypeBrowseOrCategory`

File: unbxdSearch.js [hunk 3]
```diff
@@ -339,6 +339,22 @@ var unbxdSearchInit = function(jQuery, Handlebars){
...
 
+  var isOfTypeBrowseOrCategory = function isOfTypeBrowseOrCategory(type) {
+    return (type === "browse" || type === "category");
+  };
+
+  var isCategoryIdPresent = function isCategoryIdPresent(params) {
+    return ("categoryId" in params && params.categoryId.trim().length > 0);
+  };
+
+  var isOfTypeSearch = function isOfTypeSearch(type) {
```
Comment (rahulcs) [inline unbxdSearch.js]: No need of `Of` in variable name

**Comment by santosh1994** [comment]
@rahulcs done

**Comment by santosh1994** [comment]
@rahulcs update version and README.md

**Pull request merged**